agents_infra/main.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import time
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    logging.info('agents-infra started. Press Ctrl+C to exit.')
+
+    try:
+        while True:
+            time.sleep(5)
+    except KeyboardInterrupt:
+        logging.info('Interrupting...')
+    finally:
+        logging.info('agents-infra exiting...')
```

Comment:
`logger` is declared, but `logging.info(...)` is used without configuring logging. Add a basic logging config block at the top of `main()`:
```
logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```
This ensures logs are actually emitted when running as a `systemd` service or directly.